sdcm/cluster.py

```diff
@@ -1618,6 +1618,18 @@ def patch_scylla_yaml(self, yaml_file=SCYLLA_YAML_PATH, **kwargs):
         with open(file=yaml_dst_path, mode='r') as scylla_yaml_file:
             scylla_yml = yaml.safe_load(scylla_yaml_file)
 
+        if seed_address:
+            # Set seeds
+            scylla_yml['seed_provider'] = [
+                dict(class_name='org.apache.cassandra.locator.SimpleSeedProvider',
+                     parameters=[dict(seeds=seed_address)])]
+
+            # NOTICE: the following configuration always have to use private_ip_address for multi-region to work
+            # Set listen_address
+            scylla_yml['listen_address'] = self.private_ip_address
```

Comment:
why did you put this into this function ?

Reply:
you want to set the seeds only don't change listen_address or rpc_address, scylla-cloud should handle those

Reply:
fixed